news/views.py

```
@@ -541,24 +523,91 @@ def dispatch(self, request, *args, **kwargs):
+            if image.size >= settings.DOWNSCALE_IMAGE_THRESHOLD:
+                image = downsize_uploaded_image(image)
+            wagtail_image = Image.objects.create(
+                title=image.name,
+                file=image,
+            )
+            page.image = wagtail_image
+        tags = []
+        if related_libraries:
+            for library in related_libraries:
+                lib = Library.objects.get(slug=library)
+                tag, created = ContentTag.objects.get_or_create(
```

Ruff (0.16.0)

[warning] Unpacked variable `created` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)